From 15a9cd0590cacb4b189e39a6863d6abbe9bf0a10 Mon Sep 17 00:00:00 2001
From: xiadanni1 <xiadanni1@huawei.com>
Date: Fri, 24 Jul 2020 04:20:57 +0800
Subject: [PATCH] bump to 1.13.4

Signed-off-by: xiadanni1 <xiadanni1@huawei.com>
---
 VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/VERSION b/VERSION
index 5d371fc..478bafe 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-go1.13.3
\ No newline at end of file
+go1.13.4
-- 
1.8.3.1
